Sources/Core/GTMSessionUploadFetcher.m

```diff
@@ -1513,6 +1513,11 @@ - (void)chunkFetcher:(GTMSessionFetcher *)chunkFetcher
   } else {
     // The chunk has uploaded successfully.
     int64_t newOffset = self.currentOffset + previousContentLength;
+    NSString *uploadSizeReceived = [chunkFetcher.responseHeaders objectForKey:kGTMSessionHeaderXGoogUploadSizeReceived];
```

Comment:
Please run clang-format on this (`git clang-format` should do the right thing).

Reply:
Done.